metadata.json

```
@@ -26,7 +26,7 @@
     },
     {
       "name": "camptocamp/systemd",
-      "version_requirement": ">= 0.4.0 < 3.0.0"
+      "version_requirement": ">= 0.4.0 <= 3.0.0"
```

This is usually how we do it:

```suggestion
      "version_requirement": ">= 0.4.0 < 4.0.0"
```

---

@kenyon thanks for the suggestion. Committed.